src/internal/services/auth_service.go

```
@@ -173,17 +172,24 @@ func (s *AuthService) VerifyUser(user *models.User, token string) error {
 	storedToken, err := s.AuthRepo.GetUserVerification(user.ID)
 	if err != nil {
 		if err == gorm.ErrRecordNotFound {
+
 			return errors.New("no verification token found")
 		}
 		return err
 	}
 
+	if storedToken.ExpiresAt.Before(time.Now()) {
+		s.AuthRepo.DeleteUserVerification(user.ID)
```

golangci-lint (1.64.8)

Error return value of `s.AuthRepo.DeleteUserVerification` is not checked

(errcheck)